remind.py

```diff
@@ -411,6 +411,10 @@ def to_remind(self, vevent, label=None, priority=None, tags=None, tail=None,
                 remind.append(vevent.dtstart.value.astimezone(self._localtz).strftime('AT %H:%M').replace(' 0', ' '))
             else:
                 remind.append(vevent.dtstart.value.strftime('AT %H:%M').replace(' 0', ' '))
+
+            if posttime:
```

Comment:
Indeed it would! I was not aware such a thing existed. I'll work on that.

However, I still think it would be useful to allow some sort of override, which could take one (or more) of the following forms:

- Apply to every entry that doesn't have an alarm time.
- Apply to every entry without an alarm time, plus whose alarm time is shorter than the default.
- Apply to every entry, regardless of whether it has an alarm time.

Any thoughts on the matter?